runtime/doc/lua.txt

```diff
@@ -1571,6 +1571,75 @@ schedule_wrap({cb})                                      *vim.schedule_wrap()*
       • |vim.schedule()|
       • |vim.in_fast_event()|
 
+system({spec}, {on_exit})                                       *vim.system()*
+    Run a system command
+
+    Examples: >lua
+
+      local on_exit = function(code, signal, stdout, stderr)
+        print(code)
+        print(signal)
+        print(stdout)
+        print(stderr)
+      end
+
+      -- Run asynchronously
+      vim.system({'echo', 'hello'}, on_exit)
+      vim.system({cmd = {'echo', 'hello'}}, on_exit)
+      vim.system({cmd = 'echo', args = {'hello'}}, on_exit)
+
+      -- Run synchronously
+      local obj = vim.system({'echo', 'hello'}):wait()
+      -- { code = 0, signal = 0, stdout = 'hello', stderr = '' }
+<
+
+    See |uv.spawn()| for more details.
+
+    Parameters: ~
+      • {spec}     (string[]|SystemSpec) When passed as a string array, the
+                   argument is interpreted as a command. Accepts the options:
+                   • cmd: (string|string[]) command to execute
+                   • args: (string[]) Command line arguments. Ignored when cmd
+                     is a string[]
+                   • cwd: (string) Set the current working directory for the
+                     sub-process.
+                   • env: table<string,string> Set environment variables for
+                     the new process.
+                   • stdin: (string|string[]|true) If `true`, then a pipe to
+                     stdin is opened and can be written to via the `write()`
+                     method to SystemObj.
+                   • stdout: (function|false) when passed as a function must
+                     have the signature fun(err: string, data: string) Pass
+                     `false` to disable.
+                   • stderr: (function|false) when passed as a function must
```

Comment:
These are handlers so they should use the `on_` convention, no? That's also consistent with rather strongly-established tradition for e.g. `jobstart`, and unless there's strong reason to deviate we should strongly prefer to avoid unnecessary entropy.

```suggestion
                   • on_stdout: (function|false) when passed as a function must
                     have the signature fun(err: string, data: string) Pass
                     `false` to disable.
                   • on_stderr: (function|false) when passed as a function must
```